ui/projekt-monsterz-front/src/app/login-screen/login-screen.component.css

```diff
@@ -0,0 +1,31 @@
+* {
+     padding: 5px;
+     margin: 0;
+     
+}
```

Comment:
Oh, you don't use newlines between element identifiers, huh? I'll have to get used to that.